src/lang/fr.json

```
@@ -90,7 +90,7 @@
   "ch.sbb.lar-desc": "TINA - Système d’information topographique réseau et offre Thème:<br/>Vue d’ensemble de l’offre et du réseau de l’Infrastructure incluant les différents horizons. Représentation des données prévisionnelles de l’offre (nombre de trains, matériel roulant, modifications d’un horizon à l’autre) et des projets majeurs sous forme de cartes topographiques/schématiques et de synopsis. ",
   "ch.sbb.mobz": "Mobilitätshubs",
   "ch.sbb.mobz-desc": "Mobilitätshubs",
-  "ch.sbb.netzkarte": "Carte du réseau des TP suisses",
+  "ch.sbb.netzkarte": "Carte de réseau: trafic voyageurs",
```

```suggestion
  "ch.sbb.netzkarte": "Carte du réseau: trafic voyageurs",
```
@oterral What about this?

---

Lucien said that there is only a minimal difference in meaning between "de" und "du". So, I will leave it the way the translation service provided it.